.husky/pre-commit

```diff
@@ -1,4 +1,24 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn run lint-staged
+yarn run lint
+
+if ! command -v detect-secrets-hook &> /dev/null
```

Comment:
Do we need this, or can we install it as a plugin?

It looks like it might be possible to do this entirely with config?

Reply:
I dont think we need this command to check detect secrets exists. But detect secrets needs to be installed on the users machine.

Even if using the NPM package for detect secrets we'd need to install it separately (at least that's my understanding)

https://www.npmjs.com/package/detect-secrets#:~:text=Yelp%27s%20detect%2Dsecrets%20is,with%20a%20warning%20message

I think the logic of this package is bloat as its just really a launcher